jsapp/js/components/common/audioPlayer.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react'
-import autoBind from 'react-autobind'
 import bem, {makeBem} from 'js/bem'
 import KoboRange from 'js/components/common/koboRange'
+import LoadingSpinner from 'js/components/common/loadingSpinner'
+import Icon from 'js/components/common/icon'
+import Button from 'js/components/common/button'
```

Comment:
I edited the TODOs: https://github.com/kobotoolbox/kpi/pull/3582/commits/b7d282699c39e86cae66e7da325248490218b326 the icon is smaller and the padding is bigger. I think it looks okay though, what do you think?

Reply:
It's ok :+1: I will revisit all the icons in https://github.com/kobotoolbox/kpi/pull/3635 (WIP) so if it doesn't fit the size Ignacio wanted I will change it then :)